src/liger_kernel/transformers/model/qwen2_vl.py

```diff
@@ -0,0 +1,172 @@
+from typing import List, Optional, Tuple, Union
+
+import torch
+from torch.nn import CrossEntropyLoss
+from transformers.models.qwen2_vl.modeling_qwen2_vl import (
```

Comment:
ditto. will it break users with lower version?

Reply:
Only if they explicitly import this module - and I would argue it should throw an ImportError in that case.